src/stores/uiStateStore.tsx

```diff
@@ -91,6 +92,9 @@ const initialState = () => {
         },
         setRendererEl: (el) => {
           set({ rendererEl: el });
+        },
+        setDisableScrollZoom: (disableScrollZoom) => {
```

Comment:
If you're happy with my suggestion above, we can simplify here by removing this.